_component_design/key-value-pairs.md

```diff
@@ -9,32 +9,32 @@ preview-image: preview-images/key-value.svg
 resource: true
 status: stable
 last-modified: 2017-11-07
-helix-ui-css: true
-helix-ui-javascript: true
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### When to use
+## When to use
+
+A key-value pair is a group of two related data items. Key-value pairs display a label (the key) with the corresponding data (the value). Color and alignment of key-value pairs allow the user to scan the page quickly for relevant data.
 
-Use key-value pairs when a key (label) needs either text or a visual display of that key’s corresponding value (data).
+Use key-value pairs when a key (label) needs text or a visual display of that key’s corresponding value (data).**<<not entirely sure what this means...>>**
```

Comment:
Probably best to take these types of comments outside checked in code =).

@DonNateR Wrote this content. He can help clarify.

Reply:
Would "Use key-value pairs when a key has corresponding value which can be displayed using visuals or text." 

For context @chri2547, I was trying to paraphrase the definition for Key-value pairs found on this page: http://searchenterprisedesktop.techtarget.com/definition/key-value-pair .

Reply:
👍 I will chat with Nate about the best way to handle questions I may have about content.

Updated the sentence. Thanks for the wording suggestion.